<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>372</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发 送(S)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../微信/img/search.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>492</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../微信/img/title.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="FriendNameLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>FriendName</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatTextBrowser">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>65</y>
      <width>491</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>365</y>
      <width>492</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../微信/img/tool.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>403</y>
      <width>491</width>
      <height>97</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>65</y>
      <width>250</width>
      <height>435</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userHead">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>userName</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HeadTitleBg">
    <property name="geometry">
     <rect>
      <x>743</x>
      <y>1</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>生日</string>
    </property>
   </widget>
   <widget class="QLabel" name="birthdayLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email@edu.cn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>电话</string>
    </property>
   </widget>
   <widget class="QLabel" name="PhoneLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>330</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+86XXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>156</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sex</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>个人简介</string>
    </property>
   </widget>
   <widget class="QLabel" name="introLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我是阿巴阿巴阿巴巴巴巴阿巴巴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="StateLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>42</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>FriendState</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FriendHead">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>8</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="OriginalBg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>581</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="transPushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>372</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <zorder>HeadTitleBg</zorder>
   <zorder>label_5</zorder>
   <zorder>chatTextEdit</zorder>
   <zorder>sendPushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>FriendNameLabel</zorder>
   <zorder>chatTextBrowser</zorder>
   <zorder>listView</zorder>
   <zorder>userHead</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>birthdayLabel</zorder>
   <zorder>label_6</zorder>
   <zorder>EmailLabel</zorder>
   <zorder>label_7</zorder>
   <zorder>PhoneLabel</zorder>
   <zorder>label_8</zorder>
   <zorder>sexLabel</zorder>
   <zorder>label_9</zorder>
   <zorder>introLabel</zorder>
   <zorder>StateLabel</zorder>
   <zorder>FriendHead</zorder>
   <zorder>OriginalBg</zorder>
   <zorder>transPushButton</zorder>
   <zorder>searchPushButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
